packages/server/src/cli.ts

```diff
@@ -132,13 +133,17 @@ export class Entry extends Command {
 			}
 		});
 
-		const passwordLength = 12;
-		const possible = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
-		const chars = [];
-		for (let i = 0; i < passwordLength; i++) {
-			chars.push(possible[Math.floor(Math.random() * possible.length)]);
+		let password = flags["password"];
+		if (!password) {
+			// Generate a random password
```

Comment:
Unrelated issue but this needs to be changed. The password should generated from a CPRNG.

Reply:
This PR is only related to adding a static password feature, not changing original password generation method.

Reply:
For those reading this thread, aforementioned issue is located at https://github.com/codercom/code-server/issues/26